### PR TITLE
Add configurable blur size to popup lyrics

### PR DESCRIPTION
**Pull request by ZT2wo**
More options, why not

### CODE REVIEW

File: Extensions/popupLyrics.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ if (!navigator.serviceWorker) {
     // Worker code
     // When Spotify client is minimised, requestAnimationFrame does not call our tick function
     // setTimeout and setInterval are also throttled at 1 second.
-    // Offload setInterval to a Worker to consistently call tick function.
+    // Offload setInterval to a Worker to consistenly call tick function.
```
Comment (TalEdd) [inline Extensions/popupLyrics.js:14]: Wrong spelling

Comment (TalEdd) [reply Extensions/popupLyrics.js:14]: Should be consistently

Comment (saxobroko) [reply Extensions/popupLyrics.js:14]: 'consistenly' should be 'consistently'

**Review by saxobroko — CHANGES_REQUESTED** [review]

**Comment by afonsojramos** [comment]
Merged locally due to big conflicts.

**Pull request closed without merging**